src/background/listener/bee-api.listener.ts

```diff
@@ -130,6 +134,23 @@ export class BeeApiListener {
       { urls: [`${fakeUrl.openDapp}/*`] },
     )
 
+    // 'bzz://{content-address}' URI in search bar triggers redirect to gateway BZZ address
+    // NOTE: works only if google search is set as default search engine
+    chrome.webRequest.onBeforeRequest.addListener(
+      (details: chrome.webRequest.WebRequestBodyDetails) => {
+        console.log('Original BZZ Url', details.url)
+        const urlParams = new URLSearchParams(new URL(details.url).search)
+        const query = decodeURI(urlParams.get('oq') || urlParams.get('q') || '')
+
+        if (!query || !query.startsWith('bzz://')) return
+
+        this.redirectToBzzReference(query.substr(6), details.tabId)
```

Comment:
`string.substr` is a deprecated function

Reply:
This is old code, copied.